k8s/k8s_stateful_set.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: taskbroker-service
+  labels:
+    app: taskbroker-service
+spec:
+  ports:
+  - port: 50051
+    name: grpc
+  clusterIP: None
+  selector:
+    app: taskbroker
```

Comment:
We'll also need to configure [session affinity](https://kubernetes.io/docs/reference/networking/virtual-ips/#session-affinity) for the grpc service so that workers can communicate with the same broker consistently.